client/utils.go

```diff
@@ -73,6 +86,11 @@ func ReadPageRequest(flagSet *pflag.FlagSet) (*query.PageRequest, error) {
 	}, nil
 }
 
+// ReadPageRequestWithPageKeyDecoded is a shortcut for ReadPageRequest(flagSet, FlagSetWithPageKeyDecoded)
```

Comment:
With a specific flag set decoder, yes.

The `ReadPageRequest` function reads the `--page-key` arg and puts it into the page request as if a user somehow provided the raw bytes via CLI. When a `next_key` is part of a query's output, it gets automatically converted to base64 though. That's good, but it means that, by default, you can't just copy/paste the `next_key` value into your next query as a `--page-key`.

With the bank is-sendable-denom stuff I added the `FlagSetWithPageKeyDecoded` function because everyone was having to just copy/paste that, run the flagset through it, then provide the result to `ReadPageRequest`. Originally, I wanted to call it `WithPageKeyDecoded` but they insisted on it being the longer version. The line to use it would look like this `req, err := utils.ReadPageRequest(utils.FlagSetWithPageKeyDecoded(cmd.Flags()))`.

It would have been nice to just fix `ReadPageRequest` to actually work, but everyone that uses the SDK had already copy/pasted their own `FlagSetWithPageKeyDecoded` into their repos (before it existed in the SDK), and trying to fix it in `ReadPageRequest` could easily break everyone's use of it since they all already compensate for the bug.

I made `ReadPageRequestWithPageKeyDecoded` to just be a more concise (heh) way to get what's needed. Basically, it's `req, err := utils.ReadPageRequestWithPageKeyDecoded(cmd.Flags())` now. And because of intellisense in most IDEs, it's more likely that a user might see it and realize they need to use that instead of just `ReadPageRequest`.

If I ever get this into the SDK, I'll probably add a comment to the `ReadPageRequest` function letting people know that they almost certainly want `ReadPageRequestWithPageKeyDecoded` instead.

Reply:
Oh. Heh. I forgot one major pain point (and am slightly incorrect in some of the code snippets in the previous comment). `FlagSetWithPageKeyDecoded` returns `(*pflag.FlagSet, error)`. So it _can't_ just be provided directly to `ReadPageRequest`, it has to be done in multiple steps.

So basically, this:
```golang
fs, err := utils.FlagSetWithPageKeyDecoded(cmd.Flags())
if err != nil {
    return err
}
req, err := utils. ReadPageRequest(fs)
```

is now this:
```golang
req, err := utils. ReadPageRequestWithPageKeyDecoded(cmd.Flags())
```